sites/website/src/docs/introduction.md

```diff
@@ -4,3 +4,39 @@ title: Introduction
 sidebar_label: Introduction
 custom_edit_url: https://github.com/microsoft/fast-dna/edit/master/sites/website/src/docs/introduction.md
 ---
+
+Welcome to the FAST-DNA documentation! We're glad you're here. We can't wait to show you around.
```

Comment:
I recommend we all take a collaborative spin on the documentation after choosing a voice, tone, etc.  Each of us writes with a different tone, and it'll be important to find middle ground and consistency in the final output.  These guidelines may exist on the writing page, if not, we should meet to discuss and update that page.  This is a recommendation for post relaunch.